clang/runtime/dpct-rt/include/dpct/util.hpp

```diff
@@ -658,7 +658,9 @@ namespace experimental {
 /// Note: Please make sure that all the work items of all work groups within
 /// a SYCL kernel can be scheduled actively at the same time on a device.
 template <int dimensions = 3>
-inline void nd_range_barrier(
+[[deprecated("Please use sycl::group_barrier(type of root_group) "
```

Comment:
be specific for the parameter type.

Reply:
"DEPRECATED : Experimental helper function used to help "
            "cross-group "
            "synchronization during migration. Please try to remigrate with "
            "option: --use-experimental-features=root-group\n"